gen-hash-functions-registry

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 #
-# Script to generate the multihash algorithms registry from the CSV file
+# Script to generate the multihash hash functions registry from the CSV file
```

Comment:
Hmm, so I agree that this should change... but "algorithms" nor "hash functions" are really the right thing. This is really a table of "identifiers that contain parameters to hash functions". How about this instead?

"Multihash Identifier Registry"

Reply:
If you agree, please change the rest of this PR from "multihash hash functions" to "multihash identifier registry".